### ISSUE TITLE
Document that shards of a closed index are not reported by APIs like _cat/shards

### ISSUE THREAD

**Issue by TomonoriSoejima** [elastic/elasticsearch#42334]
I ran into a situation where I lost replica shards when I closed an index and such thing is not mentioned in the doc and thought it would be nice to have a word about it.

https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-open-close.html#_wait_for_active_shards 

https://github.com/elastic/elasticsearch/issues/33888 seems to have some background as to why replica shards are removed.

**Comment by dnhatn** [comment]
@TomonoriSoejima Which ES version are you using?

**Comment by TomonoriSoejima** [comment]
I used es 6.4.0 but just tested 7.1.0 and found the same behavior.

This is what it looks in Kibana interaction.

```
PUT hello
{
  "settings": {
    "number_of_shards": 1
    , "number_of_replicas": 1
  }
  
}

GET _cat/shards?v

POST hello/_close
GET _cat/shards?v

```


```
index                shard prirep state   docs  store ip        node
.kibana_1            0     p      STARTED    4 14.5kb 127.0.0.1 node2
.kibana_1            0     r      STARTED    4 14.5kb 127.0.0.1 node1
.kibana_task_manager 0     p      STARTED    2 44.9kb 127.0.0.1 node2
.kibana_task_manager 0     r      STARTED    2   37kb 127.0.0.1 node1
hello                0     p      STARTED    0   230b 127.0.0.1 node2
hello                0     r      STARTED    0   230b 127.0.0.1 node1


{
  "acknowledged" : true
}

index                shard prirep state   docs  store ip        node
.kibana_1            0     p      STARTED    4 14.5kb 127.0.0.1 node2
.kibana_1            0     r      STARTED    4 14.5kb 127.0.0.1 node1
.kibana_task_manager 0     p      STARTED    2 45.5kb 127.0.0.1 node2
.kibana_task_manager 0     r      STARTED    2 37.5kb 127.0.0.1 node1
```

**Comment by DaveCTurner** [comment]
To be clear, the steps above show that _both_ the primary and replicas of a closed index are not reported by `GET _cat/shards`. The shard data is still there so they're not really "removed". The shards will come back when the index is opened again. I've adjusted the title of this issue.

This'll no longer be the case in 7.2.0 thanks to #33888.

**Comment by debadair** [comment]
[docs issue triage]

**Comment by jrodewig** [comment]
Closed by #50596
